src/components/movies-sorting/movies-sorting.tsx

```diff
@@ -7,12 +7,28 @@ import { Dropdown } from "../dropdown/dropdown";
 export function MoviesSorting(props: any): React.ReactElement {
 
 
+    const sortOptions = ['Title', 'Release Date'];
+
+    const onChangeSort = (option: string) => {
+        if (option === 'Title') {
+            props.setSortBy('title');
```

Comment:
more convenient strategy is to use `const SORT_OPTIONS = [ some options ]` across the components (or may be Enum if we have ts? )

Reply:
renamed